packages/components/src/Link/docs/standalone.tsx

```diff
@@ -0,0 +1,18 @@
+import { Card, Content, Footer, Link } from "@hopper-ui/components";
+import { Heading } from "react-aria-components";
+
+export default function Example() {
+    return (
+        <Card padding="core_240" UNSAFE_width="300px">
```

Comment:
i realize that in the Card's documentation, i forgot to remove the "Related components" of Header/content/footer, can you remove it in this pr plz